Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -1208,944 +1208,108 @@ class SwiftObjectServerTests: SwiftSyncTestCase {
-        let document: Document = ["name": "fido", "breed": "cane corso"]
-
-        let insertManyEx1 = expectation(description: "Insert many documents")
-        collection.insertMany([document]) { result in
-            switch result {
-            case .success(let objectIds):
-                XCTAssertEqual(objectIds.count, 1)
-            case .failure(let error):
-                XCTFail("Insert failed: \(error)")
-            }
-            insertManyEx1.fulfill()
-        }
-        wait(for: [insertManyEx1], timeout: 4.0)
-
-        collection.aggregate(pipeline: [["$match": ["name": "fido"]], ["$group": ["_id": "$name"]]]) { result in
-            switch result {
-            case .success(let documents):
-                XCTAssertNotNil(documents)
-            case .failure(let error):
-                XCTFail("Aggregate failed: \(error)")
-            }
-        }
-
-        let countEx1 = expectation(description: "Count documents")
-        collection.count(filter: document) { result in
-            switch result {
-            case .success(let count):
-                XCTAssertNotNil(count)
-            case .failure(let error):
-                XCTFail("Count failed: \(error)")
-            }
-            countEx1.fulfill()
-        }
-        wait(for: [countEx1], timeout: 4.0)
-
-        let countEx2 = expectation(description: "Count documents")
-        collection.count(filter: document, limit: 1) { result in
-            switch result {
-            case .success(let count):
-                XCTAssertEqual(count, 1)
-            case .failure(let error):
-                XCTFail("Count failed: \(error)")
-            }
-            countEx2.fulfill()
-        }
-        wait(for: [countEx2], timeout: 4.0)
-    }
-
-    func testWatch() {
-        performWatchTest(nil)
-    }
-
-    func testWatchAsync() {
-        let queue = DispatchQueue.init(label: "io.realm.watchQueue", attributes: .concurrent)
-        performWatchTest(queue)
-    }
-
-    func performWatchTest(_ queue: DispatchQueue?) {
-        let collection = setupMongoCollection()
-        let document: Document = ["name": "fido", "breed": "cane corso"]
-
-        var watchEx = expectation(description: "Watch 3 document events")
-        let watchTestUtility = WatchTestUtility(targetEventCount: 3, expectation: &watchEx)
-
-        let changeStream: ChangeStream?
-        if let queue = queue {
-            changeStream = collection.watch(delegate: watchTestUtility, queue: queue)
-        } else {
-            changeStream = collection.watch(delegate: watchTestUtility)
-        }
-
-        DispatchQueue.global().async {
-            watchTestUtility.isOpenSemaphore.wait()
-            for _ in 0..<3 {
-                collection.insertOne(document) { result in
-                    if case .failure = result {
-                        XCTFail("Should insert")
-                    }
+    func testCallFunctionResult() throws {
```

Comment:
Did this get moved? I thought we already had a test for this.

Reply:
I added a new test but the the diff makes it look like I deleted some tests, I merge it into the callFunction test